src/components/Misc/RsvProgressBar.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  current: string;
```

Comment:
For nicity:

```suggestion
  current: keyof typeof states;
```

https://www.typescriptlang.org/play?#code/MYewdgzgLgBNCGUCmEYF4YG8BQMYEsATJeALhgHIBJYpCgGlxngHMAnJJAWyTCgH0ADmxDAUEchQAKbAD8AzAK5gWDJoIA28MGHwrJAUT4B3RW3lq8eoSPbjJAVS4QkUAF7LVjPIXBJJAIIARiwowAAWGiAQLmBqAL4A3NjYUACegkgwMiCCqBiYMMBmHHzkANZIaSDyMOmZNXBQiCgw8UA

Reply:
hab ich uebernommen